tests/test_bouquets/test_bouquets.py

```diff
@@ -1,8 +1,51 @@
+import json
+from unittest.mock import patch
 from tests.base import BaseTestCase
 
+def calendar():
```

Comment:
Having this function here is not neat and is duplication. You should rather call the main function and have it behave differently for tests. This will also ensure you have good coverage of this part of the code.